.github/workflows/ci-ultra-tests.yml

```diff
@@ -146,7 +150,9 @@ jobs:
           cd ultra
           . .venv/bin/activate
           scl scenario build-all ultra/scenarios/pool
-          pytest -v ./tests/test_ultra_package.py
+          pytest -v \
+          --durations=0 \
+          ./tests/test_ultra_package.py \
 
   # test-package-via-pypi:
```

Comment:
Just curious, are we using this test anymore?

Reply:
Actually, I have uncommented it and ran the test. It works with the latest version of ultra (ultra-rl==0.2.0). We might want to keep this test to keep  a sanity check on our ultra pypi package